lib/rules/jsx-indent.js

```diff
@@ -93,46 +95,32 @@ module.exports = {
      * @param {ASTNode} node Node violating the indent rule
      * @param {Number} needed Expected indentation character count
      * @param {Number} gotten Indentation character count in the actual node/code
-     * @param {Object} loc Error line and column location
+     * @param {Boolean} isEndOfTag which is used in the fixer function
      */
-    function report(node, needed, gotten, loc) {
+    function report(node, needed, gotten, isEndOfTag) {
       var msgContext = {
         needed: needed,
         type: indentType,
         characters: needed === 1 ? 'character' : 'characters',
         gotten: gotten
       };
 
-      if (loc) {
```

Comment:
I noticed that no calls to `report` were using `loc` anyway, so I could remove `loc` and replace it with the new `isEndOfTag`